### ISSUE TITLE
Clarify the media query behavior for `(display-mode)` with `display-override`

### ISSUE THREAD

**Issue by tomayac** [WICG/beforeinstallprompt#126]
A `standalone` application returns `window.matchMedia('(display-mode: standalone)').matches` as `true`.

Will new effective display modes as values of `display_override` be detectable, too? Assuming a user agent supports `foo` mode and the manifest contains `"display_override": ["foo"]`, would then `window.matchMedia('(display-mode: foo)').matches` work?

**Comment by tomayac** [comment]
Great, opened https://github.com/w3c/manifest/issues/952 for this feature.

**Comment by kenchris** [comment]
Yes just add it to DisplayModeType and define "browser" as fallback mode when used in 'display' and not 'display_override'. That is compatible with implementations today and would expose it in the media query in implementations supporting this display mode
